docs/src/content/docs/fr/guides/pages.mdx

```diff
@@ -0,0 +1,151 @@
+
+### Ajouter des pages
+
+Ajoutez de nouvelles pages à votre site en créant des fichiers `.md` ou `.mdx` dans `src/content/docs/`.
+Utilisez des sous-dossiers pour organiser vos fichiers et pour créer plusieurs segments de chemin.
+
+Par exemple, la structure de fichier suivante générera des pages à `exemple.com/hello-world` et `exemple.com/reference/faq` :
+
+import FileTree from '~/components/file-tree.astro';
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - hello-world.md
+      - reference/
+        - faq.md
+
+</FileTree>
+
+### Sécurité de la typologie du frontmatter
+
+Toutes les pages Starlight partagent un [ensemble commun de propriétés de la page d'accueil](/fr/reference/frontmatter/) personnalisable qui permet de contrôler l'apparence de la page :
+
+```md
+---
+title: Bonjour tout le monde !
+description: Voici une page de mon site alimenté par Starlight
+---
+```
+
+Si vous oubliez quelque chose d'important, Starlight vous le fera savoir.
+
+## Pages personnalisées
+
+Pour les cas d'utilisation avancés, vous pouvez ajouter des pages personnalisées en créant un répertoire `src/pages/`.
+Le répertoire `src/pages/` utilise le [routage basé sur les fichiers d'Astro](https://docs.astro.build/fr/basics/astro-pages/#file-based-routing) et inclut le support des fichiers `.astro` parmi d'autres formats de pages.
+Ceci est utile si vous avez besoin de construire des pages avec une mise en page complètement personnalisée ou de générer une page à partir d'une source de données alternative.
+
+Par exemple, ce projet mélange du contenu Markdown dans `src/content/docs/` avec des routes Astro et HTML dans `src/pages/` :
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - hello-world.md
+  - pages/
+    - custom.astro
+    - archived.html
+
+</FileTree>
+
+Pour en savoir plus, consultez le guide ["Pages" dans la documentation Astro](https://docs.astro.build/fr/basics/astro-pages/).
+
+### Utiliser le design de Starlight dans des pages personnalisées
+
+Pour utiliser la mise en page Starlight dans des pages personnalisées, englobez le contenu de votre page avec le composant `<StarlightPage />`.
+Cela peut s'avérer utile si vous générez du contenu de manière dynamique, mais que vous souhaitez tout de même utiliser la conception de Starlight.
+
+```astro
+---
+// src/pages/custom-page/example.astro
+import StarlightPage from '@astrojs/starlight/components/StarlightPage.astro';
+import CustomComponent from './CustomComponent.astro';
+---
+
+<StarlightPage frontmatter={{ title: 'Ma page personnalisée' }}>
+	<p>Il s'agit d'une page personnalisée avec un composant personnalisé :</p>
+	<CustomComponent />
+</StarlightPage>
+```
+
+#### Props
+
+Le composant `<StarlightPage />` accepte les éléments suivants.
+
+##### `frontmatter` (required)
+
+**type :** `StarlightPageFrontmatter`
+
+Définit les [propriétés de la page d'accueil](/fr/reference/frontmatter/) pour cette page, similaire à la page d'accueil dans les pages Markdown.
```

Comment:
```suggestion
Définit les [propriétés du frontmatter](/fr/reference/frontmatter/) pour cette page, similaire au frontmatter dans les pages Markdown.
```

Any specific reason you sometime keep "frontmatter" (as expected) and sometime you translate it to "page d'accueil"?

Reply:
No, error for my part. I keep frontmatter, I just forgot to edit this part 😶